Sources/SwiftKafka/Configuration/KafkaProducerConfiguration.swift

```diff
@@ -158,6 +117,8 @@ extension KafkaProducerConfiguration: Hashable {}
 
 extension KafkaProducerConfiguration: Sendable {}
 
+extension KafkaProducerConfiguration: KafkaProducerSharedProperties {}
```

Comment:
I've tried to use at least internal shared protocol to unite mostly all properties in producer/transactional producer instead of copying them